opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -194,29 +209,25 @@ def __eq__(self, other: object) -> bool:
             return NotImplemented
         return self.__dict__ == other.__dict__
 
+    def to_dict(self) -> LogRecordDict:
+        return {
+            "body": self.body,
+            "severity_number": getattr(self.severity_number, "value", None),
```

Comment:
Yep, that seems reasonable to me. Made the change and updated the `LogRecordDict` to reflect that `severity_number` will be an `int` instead of an `Optional[int]`.